develop/parachain-devs/runtime-development/runtime-apis.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Runtime APIs
+description: Highlights the runtime interfaces that enable communication with outer node services.  These APIs allow for the outer node to communicate with the WebAssembly runtime of a Polkadot SDK-based node.
+---
+
+Nodes consist of outer node services and a runtime. This separation of responsibilities is an important concept for designing Polkadot SDK-based chains and building upgradeable logic.
```

Comment:
I think it would be really helpful to have a little picture or diagram of this concept